### PR TITLE
Feature/navbar-footer

### PR DESCRIPTION
**Pull request by Sonny97**
added the navbar and footer in gallery, quote and contact pages.

Contact:
image

Gallery: 
image

quote: 
image

### CODE REVIEW

**Comment by jusk02** [comment]
The README.md fiel should not be included here, please ensure you have it just as it is in develop

**Comment by jhonatanrojasbastidas** [comment]
the <a href> dont work  in the nav bar.

**Comment by jusk02** [comment]
Remember contact page should not have a link to contact, and the same for other pages.
Also please set the correct `href`s in each case

File: index.html [hunk 1]
```diff
@@ -75,7 +75,7 @@ <h1 class="info__title">Comodidades</h1>
           <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn3"></div>
           <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn4"></div>
         </div>
-        <button href="#" class="button button--medium button--secondary">Ver galeria</button>
+        <button href="./pages/galeria.html" class="button button--medium button--secondary">Ver galeria</button>
```
Comment (jusk02) [inline index.html]: `./pages/galeria.html` does not exist

File: pages/quote.html [hunk 2]
```diff
@@ -6,24 +6,64 @@
...
+        <div id="main-nav__burguer" class="main-nav__burguer">
+          <div class="main-nav__burguer-bar burguer__bar-1"></div>
+          <div class="main-nav__burguer-bar burguer__bar-2"></div>
+          <div class="main-nav__burguer-bar burguer__bar-3"></div>
+        </div>
+        <div id="main-nav__menu" class="main-nav__menu">
+          <ul id="main-nav__menu-items" class="main-nav__menu-items">
+            <li><a class="main-nav__menu-link" href="../index.html">Inicio</a></li>
+            <li><a class="main-nav__menu-link" href="./gallery.html">Galería</a></li>
+            <li><a class="main-nav__menu-link" href="#">Cotizar</a></li>
```
Comment (jusk02) [inline pages/quote.html]: Since this is the quote page, the link should be removed, not set to `#`

File: pages/gallery.html [hunk 3]
```diff
@@ -19,8 +19,25 @@
...
+        </div>
+        <div id="main-nav__burguer" class="main-nav__burguer">
+          <div class="main-nav__burguer-bar burguer__bar-1"></div>
+          <div class="main-nav__burguer-bar burguer__bar-2"></div>
+          <div class="main-nav__burguer-bar burguer__bar-3"></div>
+        </div>
+        <div id="main-nav__menu" class="main-nav__menu">
+          <ul id="main-nav__menu-items" class="main-nav__menu-items">
+            <li><a class="main-nav__menu-link" href="../index.html">Inicio</a></li>
+            <li><a class="main-nav__menu-link" href="#">Galería</a></li>
```
Comment (jusk02) [inline pages/gallery.html]: Since this is the gallery page, the link should be removed, not set to `#`

File: pages/contact.html [hunk 4]
```diff
@@ -6,24 +6,63 @@
...
+          <div class="main-nav__burguer-bar burguer__bar-1"></div>
+          <div class="main-nav__burguer-bar burguer__bar-2"></div>
+          <div class="main-nav__burguer-bar burguer__bar-3"></div>
+        </div>
+        <div id="main-nav__menu" class="main-nav__menu">
+          <ul id="main-nav__menu-items" class="main-nav__menu-items">
+            <li><a class="main-nav__menu-link" href="../index.html">Inicio</a></li>
+            <li><a class="main-nav__menu-link" href="./gallery.html">Galería</a></li>
+            <li><a class="main-nav__menu-link" href="./quote.html">Cotizar</a></li>
+            <li><a class="main-nav__menu-link" href="#">Contáctanos</a></li>
```
Comment (jusk02) [inline pages/contact.html]: Since this is the contact page, the link should be removed, not set to `#`

File: index.html (hunk 1 above)
Comment (Sonny97) [reply index.html]: Then where should send it?

Comment (Sonny97) [reply index.html]: :joy:

File: pages/contact.html [hunk 5]
```diff
@@ -6,24 +6,63 @@
...
     />
     <title>Contact - Hostal las paisas</title>
   </head>
   <body>
-    <nav>
-      <!-- nav bar section -->
+    <nav id="main-nav" class="main-nav">
+      <div class="container main-nav__container">
+        <div id="main-nav__logo" class="main-nav__logo">
+          <img class="main-nav__logo__image" src="./../assets/images/logo-hdlp.svg" alt="Hostal Donde Las Paisas">
```
Comment (hanna446) [inline pages/contact.html]: i think the class name is incorrect, a better approach could be `main-nav__logo-image`

File: pages/contact.html [hunk 6]
```diff
@@ -6,24 +6,63 @@
...
+          <div class="main-nav__burguer-bar burguer__bar-1"></div>
+          <div class="main-nav__burguer-bar burguer__bar-2"></div>
+          <div class="main-nav__burguer-bar burguer__bar-3"></div>
+        </div>
+        <div id="main-nav__menu" class="main-nav__menu">
+          <ul id="main-nav__menu-items" class="main-nav__menu-items">
+            <li><a class="main-nav__menu-link" href="../index.html">Inicio</a></li>
+            <li><a class="main-nav__menu-link" href="./gallery.html">Galería</a></li>
+            <li><a class="main-nav__menu-link" href="./quote.html">Cotizar</a></li>
+            <li><a class="main-nav__menu-link" >Contáctanos</a></li>
```
Comment (hanna446) [inline pages/contact.html]: please remove the space between ...`link"` and `>`

File: css/style.css [hunk 7]
```diff
@@ -57,7 +57,7 @@ video {
 	left: 20px;
 }
 
-.main-nav__logo__image {
+.main-nav__logo-image {
 	width: 89px;
```
Comment (jusk02) [inline css/style.css]: this width should be set in `main-nav__logo`, `main-nav__logo-image` class should not exist

**Comment by josemariogutierrez** [comment]
I'm seeing a problem with the logo in the pages you implemented the navbar Screen Shot 2019-07-13 at 6 16 02 PM

File: css/style.css (hunk 7 above)
Comment (camigomezdev) [reply css/style.css]: But the image has a container `main-nav__logo`.

**Comment by camigomezdev** [comment]
please replace `main-nav__logo__image` in all files.

File: css/style.css (hunk 7 above)
Comment (jusk02) [reply css/style.css]: @camigomez35 exactly, therefore `width: 89px;` should be on that container, not the image.
images should always be `width: 100%;` which is set on the global styles.

Comment (camigomezdev) [reply css/style.css]: This comes from an old PR...

**Comment by Frankruiz90** [comment]
you have conflicts

File: index.html [hunk 8]
```diff
@@ -125,7 +125,7 @@ <h1 class="info__title">Comodidades</h1>
       </a>
     </div>
     <div class="footer__disclaimer">
-      <span>Web created by </span><a class="footer__footnote" href="https://github.com/rtsc8" target="_blank">RTSC8<a />
+      <span>Web created by </span><a class="footer__footnote" href="https://github.com/rtsc8" target="_blank">RTSC8</head>a>
```
Comment (josemariogutierrez) [inline index.html]: You pushed an error here. Before pushing anything please check your code and check if everything looks correct on the browser.

File: pages/quote.html [hunk 9]
```diff
@@ -6,24 +6,64 @@
...
     />
     <title>Cotizar - Hostal las paisas</title>
   </head>
   <body>
-    <nav>
-      <!-- nav bar section -->
+    <nav id="main-nav" class="main-nav">
+      <div class="container main-nav__container">
+        <div id="main-nav__logo" class="main-nav__logo">
+          <img class="main-nav__logo__image" src="./../assets/images/logo-hdlp.svg" alt="Hostal Donde Las Paisas">
```
Comment (josemariogutierrez) [inline pages/quote.html]: You renamed the `main-nav__logo-image` class, please do it for all pages

**Comment by hanna446** [comment]
cont
gall

Vic do you think you could lower the title a little so that it looks the same as zeplin?

**Review by camigomezdev — APPROVED** [review]

**Review by jusk02 — APPROVED** [review]

**Pull request merged**